conditional/blueprints/housing.py

```diff
@@ -23,8 +26,8 @@ def display_housing():
     log.info('frontend', action='display housing')
 
     # get user data
-
     user_name = request.headers.get('x-webauth-user')
+    account = ldap_get_member(user_name)
```

Comment:
It was something I choose when transitioning over to the new library. Account is the keyword I chose to denote the passing of the actual object over a string or their username (which we called all sorts of different things [uid, username, user_name, etc]). I am not sure what you mean by integration. Even if we found a better use for the the "account" keyword, having current_account and account being completely different, unrelated possess completely different functionality would be a terrible design choice.